pkg/expect/expect_test.go

```diff
@@ -189,7 +190,7 @@ func TestSignal(t *testing.T) {
 	go func() {
 		defer close(donec)
 		err = ep.Close()
-		assert.ErrorContains(t, err, "unexpected exit code [-1] after running [/usr/bin/sleep 100]")
+		assert.Regexp(t, regexp.MustCompile(`unexpected exit code \[-1\] after running \[.*sleep 100\]`), err)
```

Comment:
```suggestion
		assert.ErrorContains(t, err, "unexpected exit code [-1]")
```
@tjungblu to confirm

Reply:
I don't mind accepting the suggestion and ignore the rest of the message :) lets see what @tjungblu says.!

Reply:
We don't care about the path... but I think that it's convenient that expect writes which command failed,
so I'm on the side of keeping check whether `sleep 100` is mentioned. 

The original change or: 
```
assert.ErrorContains(t, err, "unexpected exit code [-1]")
assert.ErrorContains(t, err, "sleep 100")
```
both works for me.

Reply:
I feel multiple asserts are better and easy to read as well, lemme adopt them instead of regex.